src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/PrisonerSearchApiClient.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client
```

Comment:
Let's move the adding of the client, prisoner numbers and prisoners into their own commit, as this is unrelated to running the service with wiremock.

Reply:
Agreed.